### DISCUSSION TITLE
Could anyone tell me : Does the quick_start.py use tree search?

### DISCUSSION THREAD

**Discussion by AeonianQuill** [deepseek-ai/DeepSeek-Prover-V1.5#8] [Q&A]
I'm puzzled by the accuracy and time duration of the responses to the questions in this file. Is it possible that tree search was not utilized?

**Comment by xinhjBrant** [comment]
No, quick_start.py is only used to verify whether the model generation pipeline is functioning correctly and does not implement a full-scale tree search.

**Reply by AeonianQuill** [reply]
> Thank u! May I make some simple modifications to this file to utilize tree search for generating answers to a particular problem? How?

**Reply by xinhjBrant** [reply]
You may replace [data_path](https://github.com/deepseek-ai/DeepSeek-Prover-V1.5/blob/main/configs/RMaxTS.py#L6) with a JSON file of your problem in the same format, and then run your command, e.g. `python -m prover.launch --config=configs/RMaxTS.py --log_dir=logs/RMaxTS_results`.
